changelog/@unreleased/pr-257.yml

```
@@ -0,0 +1,5 @@
+type: manual-task
```

manual-task suggests that users will need to do something manually, however this shouldn't impact consumers :-)

```suggestion
type: improvement
```